examples/sample_client.py

```diff
@@ -2,7 +2,12 @@
 
 from uid2_client import Uid2Client
 from uid2_client import decrypt
+from uid2_client import encrypt
+from uid2_client.identity_scope import IdentityScope
 
+# this sample client will decrypt a given advertising token into raw UID2
+# and then encrypt into a new advertising token
+# in order to show the decryption (for DSPs and Sharers) and encryption (Sharers only) functionalities
```

Comment:
#  to demonstrate decryption (for DSPs and sharers) and encryption (sharers only).

Reply:
fixed